lib/elastic_record/search_hits.rb

```diff
@@ -1,15 +1,16 @@
 module ElasticRecord
   class SearchHits
-    attr_accessor :hits, :model
+    attr_accessor :hits, :model, :count
 
     class << self
       def from_response(response)
-        new(response['hits']['hits'])
+        new(response['hits']['hits'], total: response['hits']['total'])
       end
     end
 
-    def initialize(hits)
+    def initialize(hits, total: nil)
       @hits  = hits
+      @count = total.is_a?(Hash) ? total['value'] : total
```

Comment:
I think `count` would imply that it's the number of values in the `hits` accessor, whereas `total` would imply that it's the overall count. So the accessor would need to be renamed.